crates/aitia/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+---
+default_semver_increment_mode: !pre_minor beta-dev
+---
+# Changelog
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+Initial version
```

Comment:
I think the release-automation will want the current version to be listed here so it can treat it as released. The experience around the first release of a new crate could be improved but for now I think that is still necessary